src/hooks/useInitialStartup.js

```diff
@@ -197,10 +197,10 @@ export const useInitialStartup = () => {
 
   React.useEffect(() => {
     if (initialStartup.id !== null && initialStartup.id !== '') {
+      getDiscoveryData();
       getFeedChat();
       getDomain();
       getDataFeeds();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here involve reordering the function calls within the `useEffect` hook. As long as these functions do not depend on each other, this change should not cause any issues. However, if there are dependencies between these functions, this could potentially introduce bugs. Please ensure that these functions are independent of each other before making this change.


<!-- This is an auto-generated comment by OSS CodeRabbit -->